app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,27 @@
+class SessionsController < ApplicationController
+  skip_before_action :require_login, except: [:destroy]
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    if @user = login(params[:email], params[:password])
+      redirect_back_or_to root_path
+    else
+      flash[:notice] = 'Введите правильные данные'
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.